easybuild/easyconfigs/e/ecCodes/ecCodes-2.15.0-gompi-2019b.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'ecCodes'
+version = '2.15.0'
+
+homepage = 'https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home'
+description = """ecCodes is a package developed by ECMWF which provides an application programming interface and
+ a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2,
+ WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding)."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'usempi': False}
+
+source_urls = ['https://confluence.ecmwf.int/download/attachments/45757960/']
+sources = ['eccodes-%(version)s-Source.tar.gz']
+checksums = ['9fec8ad11f380795af632fb3105c4aa37d30f22fa70dba48fd93324cf6388d59']
+
+builddependencies = [('CMake', '3.15.3')]
+
+dependencies = [
+    ('netCDF', '4.7.1'),
+    ('JasPer', '2.0.14'),
+    ('libjpeg-turbo', '2.0.3'),
+    ('libpng', '1.6.37'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = "-DENABLE_NETCDF=ON -DENABLE_PNG=ON -DENABLE_PYTHON=OFF "
+configopts += "-DENABLE_JPG=ON -DENABLE_JPG_LIBJASPER=ON "
+configopts += "-DENABLE_ECCODES_THREADS=ON"  # multi-threading with pthreads
```

Comment:
any specific reason why you did not use `ENABLE_ECCODES_OMP_THREADS` here?

Reply:
nothing else than I never tested it in this older versions, I arbitrarily started with `pthreads` until recent versions (~2.18.0) where it started to cause issues at link time and I switched to `openmp`. So it might work or it might not.